tests/data/testing_objects.py

```diff
@@ -30,39 +32,109 @@
     POSITIVE_RESULT_VALUE,
     V0_V1_CUTOFF_TIMESTAMP,
     V1_V2_CUTOFF_TIMESTAMP,
+    FIELD_MUST_SEQUENCE,
+    FIELD_PREFERENTIALLY_SEQUENCE,
+    FIELD_PROCESSED,
 )
 
 TESTING_SAMPLES: List[Dict[str, Union[str, bool]]] = [
     {
         FIELD_COORDINATE: "A01",
-        FIELD_SOURCE: "test1",
+        FIELD_SOURCE: "Test Centre",
         FIELD_RESULT: "Positive",
         FIELD_PLATE_BARCODE: "123",
         "released": True,
+        FIELD_RNA_ID: "A01aaa",
         FIELD_ROOT_SAMPLE_ID: "MCM001",
+        FIELD_MONGODB_ID: ObjectId("aaaaaaaaaaaaaaaaaaaaaaa1"),
     },
     {
         FIELD_COORDINATE: "B01",
-        FIELD_SOURCE: "test1",
+        FIELD_SOURCE: "Test Centre",
         FIELD_RESULT: "Negative",
         FIELD_PLATE_BARCODE: "123",
         "released": False,
+        FIELD_RNA_ID: "B01aaa",
         FIELD_ROOT_SAMPLE_ID: "MCM002",
+        FIELD_MONGODB_ID: ObjectId("aaaaaaaaaaaaaaaaaaaaaaa2"),
     },
     {
         FIELD_COORDINATE: "C01",
-        FIELD_SOURCE: "test1",
+        FIELD_SOURCE: "Test Centre",
         FIELD_RESULT: "Void",
         FIELD_PLATE_BARCODE: "123",
         FIELD_ROOT_SAMPLE_ID: "MCM003",
+        FIELD_RNA_ID: "C01aaa",
+        FIELD_MONGODB_ID: ObjectId("aaaaaaaaaaaaaaaaaaaaaaa3"),
     },
     {
-        FIELD_COORDINATE: "A01",
-        FIELD_SOURCE: "test1",
+        FIELD_COORDINATE: "D01",
+        FIELD_SOURCE: "Test Centre",
         FIELD_RESULT: "Positive",
         FIELD_PLATE_BARCODE: "456",
         "released": True,
         FIELD_ROOT_SAMPLE_ID: "MCM004",
+        FIELD_RNA_ID: "D01aaa",
+        FIELD_MONGODB_ID: ObjectId("aaaaaaaaaaaaaaaaaaaaaaa4"),
+    },
+]
+
+TESTING_PRIORITY_SAMPLES: List[Dict[str, Union[str, bool]]] = [
+    {
+        FIELD_ROOT_SAMPLE_ID: "MCM001",
+        FIELD_SAMPLE_ID: ObjectId("aaaaaaaaaaaaaaaaaaaaaaa1"),
+        FIELD_MUST_SEQUENCE: True,
+        FIELD_PREFERENTIALLY_SEQUENCE: False,
+        FIELD_PROCESSED: False,
+    },
+    {
+        FIELD_ROOT_SAMPLE_ID: "MCM002",
+        FIELD_SAMPLE_ID: ObjectId("aaaaaaaaaaaaaaaaaaaaaaa2"),
+        FIELD_MUST_SEQUENCE: False,
+        FIELD_PREFERENTIALLY_SEQUENCE: True,
+        FIELD_PROCESSED: False,
+    },
+    {
+        FIELD_ROOT_SAMPLE_ID: "MCM003",
+        FIELD_SAMPLE_ID: ObjectId("aaaaaaaaaaaaaaaaaaaaaaa3"),
+        FIELD_MUST_SEQUENCE: True,
+        FIELD_PREFERENTIALLY_SEQUENCE: False,
+        FIELD_PROCESSED: True,
+    },
+    {
+        FIELD_ROOT_SAMPLE_ID: "MCM004",
+        FIELD_SAMPLE_ID: ObjectId("aaaaaaaaaaaaaaaaaaaaaaa4"),
+        FIELD_MUST_SEQUENCE: False,
+        FIELD_PREFERENTIALLY_SEQUENCE: False,
+        FIELD_PROCESSED: False,
+    },
+]
+
+TESTING_PRIORITY_SAMPLES_FOR_ALDP: List[Dict[str, Union[str, bool]]] = [
```

Comment:
I'd suggest to not have centre specific tests since the logic should work on all centres. This might create confusion in future, i.e. "does ALDP work differently? Why are there tests for it and not the other centres?" etc.